userdocs/profiles.dev/docs/author-docs/dependencies.md

```diff
@@ -0,0 +1,38 @@
+---
+sidebar_position: 9
```

Comment:
i think let's place this between `Default artifact values` and `Profiles definition schema` (so position 7), wdyt?

Reply:
Applied. :)

Reply:
I have a task ticket somewhere which will make changing the order of things easier

Reply:
Yeah that would be awesome, instead of re-indexing the pages.